pkg/controller/controller_instance_test.go

```diff
@@ -92,7 +92,7 @@ func TestReconcileServiceInstanceNonExistentClusterServiceClass(t *testing.T) {
 	events := getRecordedEvents(testController)
 
 	expectedEvent := warningEventBuilder(errorNonexistentClusterServiceClassReason).msgf(
-		"References a non-existent ClusterServiceClass (ExternalName: %q) or there is more than one (found: %d)",
+		"References a non-existent ClusterServiceClass {ClassExternalName:%q} or there is more than one (found: %d)",
 		"nothere", 0,
```

Comment:
necessary to format the `0`, or can you put it into the string? same with a few below